### DISCUSSION TITLE
Integration with Polygon-Edge with blockscout explorer create issues. Could you please help me out?

### DISCUSSION THREAD

**Discussion by isingh108** [0xPolygon/polygon-edge#1072] [Q&A]
# Blockscout doesn't show the creation code and bytecode of any smart contract
## Description
We have been trying to run a Polygon-edge chain with Blockscout as our block explorer since past 2 months. Blockscout works perfectly with Geth based chains.
 
The issue is that blockscout is unable to display the creation code and the bytecode for any smart contract.

Previously, whenever we tried to fetch a smart contract's code, the chain returned "unable to fetch code" error messages. This seems to have been fixed by the v0.6.2 release.

Is it due to the differences in RPC protocols of geth and polygon-edge? Do we need to go with polygon-supernets to build a production system?

## Your environment
- Ubuntu 20.04 LTS.
- v0.6.2 (unable to find "polygon-edge" in node's docker container)
- branch: develop.
- Cloud hosted: AWS.
- Docker Compose.

## Steps to reproduce
- setup a polygon-edge using docker compose cloud deployment.
- Run blockscout using docker compose after defining envirnoment variables

## Expected behavior
- Blocksouct should show the creation code and bytecode of smart contracts

## Proposed solution
The blockscout tutorial recommends the usage of Trapesys fork of blockscout. We tried it with the previous version of polygon-edge but that also showed identical errors/behavior.

**Comment by isingh108** [comment]
@laviniat1996 Hi Any suggestions or solutions on issue i'm facing integration of polygon-edge chain with blockscout explorer for production.

**Comment by ivanbozic21** [comment]
We're waiting for your response under the issue section.

**Comment by isingh108** [comment]
Hi @ivanbozic21 My problem is solved thanks.

**Reply by ghost** [reply]
How did you solve it? I have the exact same issue. Thanks.
